src/client/components/MovieCard/index.test.jsx

```diff
@@ -0,0 +1,6 @@
+
+describe('MovieCard index.jsx', () => {
```

Comment:
Please clean up.

Reply:
do you mean first empty line?
if yes, then done.

Reply:
This file is useless :)

Reply:
yes, I agree.